### PR TITLE
Breaking change script

### PR DESCRIPTION
**Pull request by wyunchi-ms**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by isra-fel** [comment]
Hi @wyunchi-ms , does this pr fix any issue? Should it be reviewed? Thanks

File: tools/StaticAnalysis/Program.cs [hunk 1]
```diff
@@ -49,7 +49,7 @@ public static void Main(string[] args)
             Console.WriteLine("Analyzer invoked with parameters: {0}", string.Join(" ", args));
             try
             {
-                string installDir = null;
+                string installDir = "C:\\Users\\yunwang\\source\\repos\\azure-powershell\\artifacts\\Debug";
```
Comment (Skuldo) [inline tools/StaticAnalysis/Program.cs]: This has a reference to a specific home directory & isn't generic.

File: tools/StaticAnalysis/Program.cs [hunk 2]
```diff
@@ -85,6 +85,7 @@ public static void Main(string[] args)
                     reportsDirectory = args[idx + 1];
                     logReportsDirectoryWarning = false;
                 }
+                reportsDirectory = "C:\\Users\\yunwang\\source\\repos\\azure-powershell\\artifacts\\StaticAnalysisResults";
```
Comment (Skuldo) [inline tools/StaticAnalysis/Program.cs]: This has a reference to a specific home directory & isn't generic.

File: tools/StaticAnalysis/Program.cs (hunk 1 above)
Comment (wyunchi-ms) [reply tools/StaticAnalysis/Program.cs]: Hi @Skuldo, thanks for your contribution. This is just a draft PR. But I will fix this when other logic is completed.

**Comment by isra-fel** [comment]
I think the work has been done seperately. I'll close this PR.

**Pull request closed without merging**